source/php.txt

```diff
@@ -23,15 +23,23 @@ MongoDB PHP Driver
 Introduction
 ------------
 
-The PHP driver consists of two components, the MongoDB `extension <https://github.com/mongodb/mongo-php-driver>`_
-and `library <https://docs.mongodb.com/php-library/current>`_.
-
-The extension provides a low-level API and mainly serves to integrate
-`libmongoc and libbson <https://docs.mongodb.com/drivers/c/>`_ with PHP.
-
-While it is possible to use the extension alone, users are strongly encouraged
-to use the extension and library together. The library provides a high-level API
-consistent with other MongoDB language drivers.
+Welcome to the documentation site for the official MongoDB PHP driver. 
+You can add the driver to your application to work with MongoDB in PHP. 
+The MongoDB PHP Driver consists of the two following components:
+
+- The `extension <https://github.com/mongodb/mongo-php-driver>`_, which 
+  provides a low-level API and mainly serves to integrate 
+  `libmongoc and libbson <https://docs.mongodb.com/drivers/c/>`_ with 
+  PHP.
+
+- The `library <https://docs.mongodb.com/php-library/current>`_, which 
+  provides a high-level API for working with MongoDB 
+  databases consistent with other MongoDB language drivers.
```

Comment:
Slightly unrelated to your PR, but I wonder if "consistent with other MongoDB language drivers." offers any value to anyone not familiar with one of the other language drivers already.

Reply:
It doesn't add much, but I do think it paints a slightly clearer picture of what the library component is and the value it adds on top of the extension. It indicates how high a level the "high-level" API is - on par with other official drivers.